src/api/dl-tal-community/content-types/dl-tal-community/schema.json

```diff
@@ -0,0 +1,27 @@
+{
+  "kind": "collectionType",
+  "collectionName": "dl_tal_communities",
+  "info": {
+    "singularName": "dl-tal-community",
+    "pluralName": "dl-tal-communities",
+    "displayName": "DLTalCommunity"
+  },
+  "options": {
+    "draftAndPublish": true
+  },
+  "pluginOptions": {},
+  "attributes": {
+    "name": {
+      "type": "string"
+    },
+    "emailID": {
```

Comment:
Does someone need to have an account with Dev Launchers' website to join the talent community? If so, we can link to their user profile https://github.com/dev-launchers/strapiv4/blob/main/src/extensions/users-permissions/content-types/user/schema.json#L22

Reply:
Discussed offline, people don't need to have an account to use this feature.